src/runtime/virtcontainers/utils/retry/retry.go

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (c) 2020. Ant Group. All rights reserved.
```

Comment:
Please confirm copyright again, if this file is copied from another project, it's better to clear where it is from.

Reply:
yeah, I have add `copy from`